modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -9235,6 +9235,7 @@ languages-to-translate-to=Languages to Translate To
 languages-translated-into=Languages Translated Into
 languages-with-override=Languages With Override
 lar-build-number-x-does-not-match-portal-build-number-x=LAR build number {0} does not match portal build number {1}.
+lar-expiry-period-alert-message=Export Entries and Their Respective LAR Files Will Be Removed After 30 Days
```

Comment:
Hi @thabet91,
I think this change should be committed as part of [LPS-166513 adding alert message for export expiry](https://github.com/tototrinh/liferay-portal/pull/55/commits/2c43b5e274ffa506f9b9463833c6155d860179bb).
And is there a reason to capitalize every first letter of every word?

Reply:
@tototrinh I will make the changes regarding the commit part. As for the capitalization of the first letter I remember BChan told us to do so but now I remember that only applies to titles. I will change it.

Reply:
For this
```
lar-expiry-period-alert-message=Export Entries and Their Respective LAR Files Will Be Removed After 30 Days
```
I'd want to share some concerns from what i have experienced when it comes to English writing, especially when submitting the PR to product team:
- `expiry` doesn't exist as a term being used elsewhere in Liferay, then we might be asked to explain "why do we need to introduce this new term"?
- This message is not mainly talking about "LAR file", it's talking mainly about the Export, so use a message key as `lar-...` may not be accepted.
- "Export Entries" will also be asked, where is the existing usage/definition of the term "Export Entry"? So i feel like simply use "Exports" would be safer.
- Finally, this is a sentence which explains something. Look 1 line up in this file, the sentence should have most of its normal words as lowercase. For capitalized text like we're using, we would only use if this sentence is used as a TITLE of a section or LABEL of a field.

That's what i recommend to double check, i won't judge much but just warn you guys so that it won't take much time of the product team to ask for approval from BChan ;-)

Reply:
Thanks @hhtank for the helpful suggestions.
@thabet91 please take a look at these. Thanks.

Reply:
Hi @thabet91, I think the message could be changed to
`export-processes-will-be-deleted-after-30-days=Export processes and export LAR files will be deleted after 30 days.`
What do you think? Thanks.

Reply:
@tototrinh how about this `export-entries-deletion-alert-message=Export entries and their respective LAR files will be removed after 30 days.`

Reply:
@thabet91, I'm not sure. As mentioned by @hhtank, I've found nowhere to use `Export Entries` and `respective LAR files`. But it looks like there are some examples of `Export LAR files `and `Export Processes`. See:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties#LL5312C1-L5312C138
https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties#L824

Reply:
@tototrinh we can go with what you suggested above then.
do you want me to make the change?

Reply:
Yes, sure. Please help change it. Thanks.

Reply:
@tototrinh but I will be putting the message key as `export-process-deletion-alert-message`

Reply:
@tototrinh that works too. Let me make the changes

Reply:
@tototrinh I have made the changes

Reply:
@thabet91 it should be:
`export-process-deletion-warning-message=Export process and export LAR file will be deleted after 30 days.`
or 
`export-processes-deletion-warning-message=Export processes and export LAR files will be deleted after 30 days.`
I guess in this case, it should be `export-process-deletion-warning-message=Export process and export LAR file will be deleted after 30 days.`
Thanks.

Reply:
@tototrinh you mean "processes" and "files" should be singular not plural?

Reply:
If I understand correctly, this message is a warning for the export process we are working on. So I think we should use singular.

Reply:
@tototrinh please check now